apps/web/src/pages/Home/ExecutiveHomePage/ExecutiveHomePage.tsx

```diff
@@ -0,0 +1,523 @@
+            id: "oQR/Ac",
+          })}
+        </p>
+        <Heading level="h3">
+          {intl.formatMessage({
+            defaultMessage:
+              "Learn more about our 4 services and discover how we can help you achieve your goals!",
+            id: "02DYRh",
+            description: "Heading for exposition services",
+          })}
+        </Heading>
+        <Accordion.Root
+          mode="simple"
+          type="single"
+          collapsible
+          data-h2-margin="base(x1 0)"
+          data-h2-display="base(flex)"
+          data-h2-flex-direction="base(column)"
+          data-h2-gap="base(x1 0)"
+        >
+          <Accordion.Item
+            data-h2-radius="base(s)"
+            data-h2-shadow="base(medium)"
+            value="digital-community-support"
+          >
+            <StandardAccordionHeader
+              headingAs="h4"
+              data-h2-padding="base(x.5 x.25)"
+            >
+              {intl.formatMessage({
+                defaultMessage: "Digital community support",
+                id: "pHIcBU",
+                description:
+                  "Button text to open section describing digital community support",
+              })}
+            </StandardAccordionHeader>
+            <Accordion.Content>
+              <p>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "Conducted 573 personalized 1:1 career management and support meetings with aspiring executives and executives in the 2022-2023 fiscal year.",
+                  id: "DsUtfO",
+                  description: "Description of digital community support",
+                })}
+              </p>
+            </Accordion.Content>
+          </Accordion.Item>
+          <Accordion.Item
+            data-h2-radius="base(s)"
+            data-h2-shadow="base(medium)"
+            value="id-digital-talents"
+          >
+            <StandardAccordionHeader
+              headingAs="h4"
+              data-h2-padding="base(x.5 x.25)"
+            >
+              {intl.formatMessage({
+                defaultMessage:
+                  "Identification of digital talents - Talent management round table discussions",
+                id: "BJFLx/",
+                description:
+                  "Button text to open section describing identification of digital talents",
+              })}
+            </StandardAccordionHeader>
+            <Accordion.Content>
+              <p>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "Across the years, hundreds of nominees at different groups and levels (EXposition focus on EX while EXtend EXposition focus on the non-EX groups and levels) have been brought forward and their nominations are compiled in the Digital Talent Management Repository. This Talent Management repository is the primary data source when organizations require to obtain Referrals throughout the year.",
```

Comment:
Thanks 👍 Should all be in [fix typos, grammar](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7294/commits/61afbdfe2c4b55f12f22793ff6e0a33cd8fc1eaf)

Reply:
@gobyrne can you confirm these changes are ok to make?

Reply:
Hmm, would this work: "...when organizations require referrals...". It doesn't read well when we combine "require" and "obtain" since it sounds like they are used interchangeably here as well as being used at the same time.

Reply:
I am okay with these changes @mnigh & @esizer ... I think let's leave the last one as is for now - and we can check in with Anouk about the language here.